packages/api/lib/testUtils.js

```diff
@@ -620,20 +620,14 @@ const createAsyncOperationTestRecords = async (context) => {
   };
 };
 
+// this doesn't look like it's being used anywhere, should I remove it?
```

Comment:
If it's not used, I'd say yes. It's an exported member so technically (possibly) a user-facing change for those using this package but... it's only test utils.

Reply:
removed as it was not used anywhere I could find